Alderis/ColorPickerSlidersViewController.swift

```diff
@@ -141,139 +152,49 @@ class ColorPickerSlidersViewController: UIViewController, ColorPickerTabProtocol
 			mainStackView.bottomAnchor.constraint(greaterThanOrEqualTo: view.bottomAnchor, constant: -15),
 			topSpacerView.heightAnchor.constraint(equalToConstant: 0),
 			hexTextField.widthAnchor.constraint(equalToConstant: 100),
-			hexStackView.heightAnchor.constraint(equalTo: anyColorRow.heightAnchor)
+			hexStackView.heightAnchor.constraint(equalTo: colorRows[0].heightAnchor)
 		])
 
 		updateMode()
 	}
 
-	override func viewDidLayoutSubviews() {
```

Comment:
@kirb is there a reason that `updateColor` was being called in `viewDidLayoutSubviews`? Removing it resolves #2 but I hope that doesn't lead to a regression.